project/Dependencies.scala

```diff
@@ -50,7 +50,11 @@ object Dependencies {
    * @see https://jena.apache.org/
    * @see http://repo1.maven.org/maven2/org/apache/jena
    */
-  val jenaLibs = "org.apache.jena" % "apache-jena-libs" % "3.1.1"
+// this does not work with  version 1.0.0 of Ammonite
+// https://github.com/lihaoyi/Ammonite/issues/664
```

Comment:
This issue in Ammonite is closed, and importing works. Should this be updated here as well?

Reply:
yep, was just doing that. Then I realised that I can't compile with JDK 9.
So I am checking what I can do to add jkd 9 support now... (Any tips anyone?)

Reply:
Perhaps it's not worth it until this bug is solved: https://github.com/sbt/sbt/issues/3631

Reply:
I would think a jdk9-update warrants its own PR..